.mocharc.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+
+const nodeOptions = ["experimental-vm-modules"];
+const [major, _minor, _patch] = process.versions.node.split('.');
+if (Number(major) >= 23) {
+  nodeOptions.push('no-experimental-strip-types');
+}
+
+module.exports =
+{
```

Comment:
The assignment being on a newline stood out to me as odd 😅 this probably isn't picked up by eslint, can you just run prettier on this file manually